### PR TITLE
Remove non-existing files

### PR DESCRIPTION
**Pull request by fernandog**
@OmgImAlexis 
```
2016-08-23 17:25:02 WARNING  TORNADO :: [b2476ac] 404 GET /js/core.min.js?12168 (143.107.176.135) 3.16ms
2016-08-23 17:25:01 WARNING  TORNADO :: [b2476ac] 404 GET /images/donate.jpg (143.107.176.135) 2.94ms
2016-08-23 17:25:01 WARNING  TORNADO :: [b2476ac] 404 GET /js/core.min.js?12168 (143.107.176.135) 5.72ms
```

### CODE REVIEW

**Comment by fernandog** [comment]
@OmgImAlexis I still get:

```
2016-08-23 21:03:11 WARNING  TORNADO :: [5715e23] 404 GET /favicon.ico (192.168.1.2) 12.36ms
```

**Comment by fernandog** [comment]
@OmgImAlexis anything else you think we should add/remove in apibuilder.mako ?

File: gui/slick/views/apiBuilder.mako [hunk 1]
```diff
@@ -57,9 +42,9 @@
     <link rel="stylesheet" type="text/css" href="css/browser.css?${sbPID}" />
     <link rel="stylesheet" type="text/css" href="css/lib/jquery-ui-1.10.4.custom.min.css?${sbPID}" />
     <link rel="stylesheet" type="text/css" href="css/style.css?${sbPID}"/>
-    <link rel="stylesheet" type="text/css" href="css/${sbThemeName}.css?${sbPID}" />
+    <link rel="stylesheet" type="text/css" href="css/${sickbeard.THEME_NAME}.css?${sbPID}" />
 </head>
-<body>
+<body ${('data-controller="' + controller + '" data-action="' + action + '" api-key="' + sickbeard.API_KEY +'"  api-root="api/v2/"', '')[title == 'Login']}>
```
Comment (fernandog) [inline gui/slick/views/apiBuilder.mako]: @OmgImAlexis what this does?

Comment (OmgImAlexis) [inline gui/slick/views/apiBuilder.mako]: This is needed for core.js to load the info params. The same is on the main.mako file.

**Comment by OmgImAlexis** [comment]
Approved!

**Pull request merged**